### PR TITLE
preparing documents for 2020Q4

### PR DESCRIPTION
**Pull request by stuij**

### CODE REVIEW

File: aapcs64-morello/aapcs64-morello.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@
    See LICENSE file for details
 
 .. |release| replace:: 2020Q4
-.. |date-of-issue| replace:: 9\ :sup:`th` December 2020
+.. |date-of-issue| replace:: 21\ :sup:`th` December 2020
```
Comment (philgee-oss) [inline aapcs64-morello/aapcs64-morello.rst:7]: :sup:`st`

File: cppabi32/cppabi32.rst [hunk 2]
```diff
@@ -4,22 +4,22 @@
    See LICENSE file for details
 
 .. |release| replace:: 2020Q4
-.. |date-of-issue| replace:: 11\ :sup:`th` November 2020
+.. |date-of-issue| replace:: 21\ :sup:`th` December 2020
```
Comment (philgee-oss) [inline cppabi32/cppabi32.rst:7]: :sup:`st`

**Review by philgee-oss — COMMENTED** [review]
General comment for all documents
---------------------------------
The main chapter titles have a mix of uppercase and lowercase. They should all be sentence case except where they would normally be uppercase. 


aadwarf64-morello
-----------------
Should the change history, include the date?

aaelf64
-------
Change history table requires an entry for 21st December 2020.

aapcs64-morello
---------------
Change history needs to include the date

aapcs64
-------
Does the change history table require an entry for 21st December 2020?

addenda32
---------
Does the change history table require an entry for 21st December 2020?

clibabi32
---------
Does the change history table require an entry for 21st December 2020?

cppabi32
--------
Change history shows 21th :sup:`th` December 2020. Should be 21 :sup:`st`.

ehabi32
-------
Does the change history table require an entry for 21st December 2020?

pauthabielf64
-------------
Does the change history table require an entry for 21st December 2020?

vfabia64
--------
Does the change history table require an entry for 21st December 2020?

File: aapcs64-morello/aapcs64-morello.rst (hunk 1 above)
Comment (stuij) [reply aapcs64-morello/aapcs64-morello.rst:7]: done

File: cppabi32/cppabi32.rst (hunk 2 above)
Comment (stuij) [reply cppabi32/cppabi32.rst:7]: done

**Review by stuij — COMMENTED** [review]
I believe I addressed all the comments except regarding the documents who's change-history table hasn't been updated:
These documents haven't changed. But I included them in this release as this makes it easier for people to understand what is and get to the latest release of each document on Github. We did the same for the previous release.

File: aadwarf64-morello/aadwarf64-morello.rst [hunk 3]
```diff
@@ -192,7 +192,7 @@ Change history
 +=========+=====================================+==============================+
 | 00alpha | September 2020                      | Alpha release.               |
 +---------+-------------------------------------+------------------------------+
-| 2020Q4  | December 2020                       | Document released on Github. |
+| 2020Q4  | 21st December 2020                  | Document released on Github. |
```
Comment (philgee-oss) [inline aadwarf64-morello/aadwarf64-morello.rst:195]: 21\ :sup:`st`

File: aaelf32/aaelf32.rst [hunk 4]
```diff
@@ -375,7 +375,7 @@ Change history
   |       |                     | In `ELF Identification`_, added        |
   |       |                     | EI_OSABI value for ELFOSABI_ARM_FDPIC. |
   +-------+---------------------+----------------------------------------+
-  | 2020Q4| 6th November 2020   | - document released on Github          |
+  | 2020Q4| 21st December 2020  | - document released on Github          |
```
Comment (philgee-oss) [inline aaelf32/aaelf32.rst:378]: 21\ :sup:st

File: aaelf64-morello/aaelf64-morello.rst [hunk 5]
```diff
@@ -193,7 +193,7 @@ Change history
   +===============+====================+=========================================+
   | 00alpha       | 1st October 2020   | Alpha release.                          |
   +---------------+--------------------+-----------------------------------------+
-  | 2020Q4        | 5th November 2020  | Document released on Github.            |
+  | 2020Q4        | 21st December 2020 | Document released on Github.            |
```
Comment (philgee-oss) [inline aaelf64-morello/aaelf64-morello.rst:196]: 21\ :sup:st

File: aapcs64-morello/aapcs64-morello.rst [hunk 6]
```diff
@@ -188,7 +188,7 @@ Change history
     +==========+=====================+========================================+
     | 00alpha  | 1st October 2020    | Alpha release.                         |
     +----------+---------------------+----------------------------------------+
-    | 2020Q4   | December 2020       | Document released on Github.           |
+    | 2020Q4   | 21st December 2020  | Document released on Github.           |
```
Comment (philgee-oss) [inline aapcs64-morello/aapcs64-morello.rst:191]: 21\ :sup:st

File: semihosting/semihosting.rst [hunk 7]
```diff
@@ -200,7 +200,7 @@ Change history
   |         |                  | example, change `_` to `-` in                        |
   |         |                  | `:semihosting-features`.                             |
   +---------+------------------+------------------------------------------------------+
-  |  2020Q4 | 10 December 202  | - document released on Github                        |
+  |  2020Q4 | 21 December 2020 | - document released on Github                        |
```
Comment (philgee-oss) [inline semihosting/semihosting.rst:203]: 21\ :sup:`st`

**Review by philgee-oss — COMMENTED** [review]
A few minor changes.

**Comment by stuij** [comment]
I fixed the lastest review comments.

Also, I forgot to address the top comment about the case of chapter titles:

I gave this a shot, but unfortunately this issue isn't totally trivial. The chapter title names are used as anchors and ids to jump to from other parts of the documents. So these references would also need updating. This on top of the fact that we would need change quite a few documents, a lot of which are fairly large.

Taken together with the fact that this mixing of case conventions has been present in the ABI for almost two decades, I think it would make more sense to open a Github issue, and fix this in the next ABI.

File: rtabi32/rtabi32.rst [hunk 8]
```diff
@@ -257,7 +257,7 @@ Change history
   |            |                               | types`_, specified handling of infinity and NaN in      |
   |            |                               | f2h_alt and d2h_alt.                                    |
   +------------+-------------------------------+---------------------------------------------------------+
-  | 2020Q4     | 6\ :sup:`th` November 2020    | - document released on Github                           |
+  | 2020Q4     | 21\ :sup:`th` December 2020   | - document released on Github                           |
```
Comment (philgee-oss) [inline rtabi32/rtabi32.rst:260]: 21\ :sup:`st`

File: aadwarf32/aadwarf32.rst [hunk 9]
```diff
@@ -218,7 +218,7 @@ Change history
   | 2018Q4 | 21\ :superscript:`st` December 2018 | Minor typographical fixes, updated     |
   |        |                                     | links.                                 |
   +--------+-------------------------------------+----------------------------------------+
-  | 2020Q4 | 23\ :superscript:`th` November 2020 | - document released on Github          |
+  | 2020Q4 | 21\ :superscript:`st` December 2020 | - document released on Github          |
```
Comment (philgee-oss) [inline aadwarf32/aadwarf32.rst:221]: Should this be 21\ :sup:`th`

**Review by philgee-oss — COMMENTED** [review]
Nearly there.

**Review by philgee-oss — APPROVED** [review]
LGTM

**Pull request merged**